setup.cfg

```diff
@@ -34,6 +34,3 @@ where=src
 [options.extras_require]
 tzdata =
     tzdata @ git+https://github.com/pganssle/tzdata.git
-
-[bdist_wheel]
-universal=1
```

Comment:
What else was this doing?  If it is really obsolete, it should go in now, as part of a separate bugfix.

Reply:
Setting `universal=1` tells `wheel` to build a universal wheel (e.g. not an architecture-specific one). Once you add a C extension, separate wheels need to be built for each platform, and it is no longer appropriate to build universal wheels.